ui/src/app/pages/workspace/workspace-edit.spec.tsx

```diff
@@ -392,13 +381,23 @@ describe('WorkspaceEdit', () => {
     const validInput = fp.repeat(100, 'a');
     wrapper.find('[data-test-id="other-specialPopulation-text"]').first().simulate('change', {target: {value: validInput}});
 
-    let otherSpecificPopulationError = getSaveButtonDisableMsg(wrapper, 'otherSpecificPopulation');
-    expect(otherSpecificPopulationError).toBeUndefined();
+    expect(getSaveButtonDisableMsg(wrapper, 'otherPopulationDetails')).toBeUndefined();
 
     const inValidInput = fp.repeat(101, 'a');
     wrapper.find('[data-test-id="other-specialPopulation-text"]').first().simulate('change', {target: {value: inValidInput}});
 
-    otherSpecificPopulationError = getSaveButtonDisableMsg(wrapper, 'otherSpecificPopulation');
-    expect(fp.first(otherSpecificPopulationError)).toBe('Other specific population must be true');
+    expect(getSaveButtonDisableMsg(wrapper, 'otherPopulationDetails')).toBeDefined();
+  });
+
+  it ('should show error message when other disseminate checked but empty', async() => {
+    const wrapper = component();
+    wrapper.find('[data-test-id="OTHER-checkbox"]').at(1).simulate('change', { target: { checked: true } });
```

Comment:
I wish our data-test-ids were a bit more consistent in casing/style, but that's a problem for another PR